File: rest/result.py
import json
class Result():

    _event_id = 0;
    _member_id = 0;
    _score = 0;

    def __init__(self, event_id, player_id, score):
        self._event_id = event_id
        self._player_id = player_id
        self._score = score

    def jsonify(self):
        data = { 'event_id' : self._event_id,
                 'member_id' : self._member_id,
                 'score' : self._score }
        return json.dumps(data)


    @property
    def event_id(self):
        return self._event_id

    @property
    def member_id(self):
        return self._member_id

    @property
    def score(self):
        return self._score

File: app/static/result.js
function Result(event_id, member_id, result_score) {
  this.event_id = event_id;
  this.member_id = member_id;
  this.result_score = result_score;

  this.get_member_id = function() {
    return this.member_id;
  }
  this.get_event_id = function() {
    return this.event_id;
  }
  this.get_result_score = function() {
    return this.result_score;
  }
}
